R/data-objects.R

```diff
@@ -144,6 +236,70 @@ iget <- function(
   # expand logical path to absolute logical path
   logical_path <- get_absolute_lpath(logical_path)
 
+  # handle iRODS to local file conversion
+  resp <- irods_to_local(
+    logical_path = logical_path,
+    offset = offset,
+    count = count,
+    verbose = verbose
+  )
+
+  # check for local file
+  stop_local_overwrite(overwrite, path)
+
+  # close connection on exit
+  on.exit(close(con))
+
+  # write file
+  con <- file(path, "wb")
+  writeBin(resp, con)
+
+  # return
+  invisible(NULL)
+}
+
+#' @rdname isaveRDS
+#'
+#' @export
+ireadRDS <- function(
+    logical_path,
+    path = ".",
```

Comment:
There shouldn't be a `path` here, since this function should only take an iRODS path to an RDS file and return the contents as an object, right?

Reply:
It's also not used

Reply:
agreed

Reply:
Forgot to remove it.